frontend/src/lib/components/QuestionOptions.svelte

```diff
@@ -8,37 +11,105 @@
+        {:else if incorrect.includes(index)}
+            <div class="m-3 w-1/2 rounded-lg bg-red-200 p-3">
+                <DisplayOption {option} {index} correct={question.correctAnswer} checked={false} />
+            </div>
+
+            <!-- this div is displayed if the form has not been submitted and the option is selected -->
+        {:else if selected_answer.includes(index)}
+            <div class="m-3 w-1/2 rounded-lg bg-slate-300 p-3">
+                <DisplayOption {option} {index} correct={question.correctAnswer} checked={true} />
+            </div>
+
+            <!-- this div is displayed if the form has not been submitted and the option is not selected yet -->
         {:else}
-            <input type="radio" id={index.toString()} bind:group={selected_answer} value={index} />
+            <div class="m-3 w-1/2 rounded-lg p-3 hover:bg-slate-300">
+                <DisplayOption {option} {index} correct={question.correctAnswer} checked={false} />
+            </div>
         {/if}
-        <label for={index.toString()}>{option}</label>
-        <br />
     {/each}
-    <button type="submit">Submit</button>
+
+    <!-- this is the submit button -->
+    <button
+        class="m-3 inline-flex items-center gap-x-2 rounded-lg border border-gray-200 px-4 py-3 text-sm font-semibold text-gray-500 hover:border-blue-600 hover:text-blue-600 disabled:pointer-events-none disabled:opacity-50 dark:border-gray-700 dark:text-gray-400 dark:hover:border-blue-600 dark:hover:text-blue-500 dark:focus:outline-none dark:focus:ring-1 dark:focus:ring-gray-600"
+        type="submit">Submit</button
+    >
+    <br />
+
+    <!-- this is the back button -->
+    <button
+        class="m-3 inline-flex items-center gap-x-2 rounded-lg border border-gray-200 px-4 py-3 text-sm font-semibold text-gray-500 hover:border-blue-600 hover:text-blue-600 disabled:pointer-events-none disabled:opacity-50 dark:border-gray-700 dark:text-gray-400 dark:hover:border-blue-600 dark:hover:text-blue-500 dark:focus:outline-none dark:focus:ring-1 dark:focus:ring-gray-600"
+        ><svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke-width="1.5"
+            stroke="currentColor"
+            class="h-6 w-6"
+        >
+            <path
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                d="M6.75 15.75 3 12m0 0 3.75-3.75M3 12h18"
+            />
+        </svg>
+    </button>
+
+    <!-- this is the next button -->
+    <button
+        class="m-3 inline-flex items-center gap-x-2 rounded-lg border border-gray-200 px-4 py-3 text-sm font-semibold text-gray-500 hover:border-blue-600 hover:text-blue-600 disabled:pointer-events-none disabled:opacity-50 dark:border-gray-700 dark:text-gray-400 dark:hover:border-blue-600 dark:hover:text-blue-500 dark:focus:outline-none dark:focus:ring-1 dark:focus:ring-gray-600"
```

Comment:
Since we created a `navigation-button-style` above, you can reference it, since they are the same style. Remember to import the pcss in the script tag.